### PR TITLE
Add payment tester option

### PR DESCRIPTION
**Pull request by Jalepeno112**
Facebook has this concept of "Payment Testers."  These are registered developers, admins or users associated with an app.  Any payment that these users do within the app is always a test payment.  That way, you don't have to set and manage the `is_test_payment` flag.  It is also the only way to test payments in Messenger Extensions.

The only way to manage these payment testers right now is through an API call to Facebook.  I've added an UI element so that people can manage these users from within Botpress.

There are several other [payment options for Messenger](https://developers.facebook.com/docs/messenger-platform/thread-settings/payment) that I want to bring into Botpress, so I created a new section in the UI specifically dedicated to it.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -50,6 +54,9 @@ const initializeMessenger = (bp, configurator) => {
   .then(config => {
     messenger = new Messenger(bp, config)
 
+    users = Users(bp, messenger);
+    bp.logger.info(users);
```
Comment (slvnperron) [inline src/index.js]: is this a debug leftover?

File: src/index.js [hunk 2]
```diff
@@ -218,6 +226,24 @@ module.exports = {
         res.send({ homepage: packageJSON.homepage })
       })
 
+      router.get("/users", (req, res)=> {
+        const user_promise = users.getAllUsers()
+          .then((values) => {
+            res.send(values)
+          })
+          .catch((err) => res.status.send(500).send({message:err.message}))
```
Comment (slvnperron) [inline src/index.js]: sorry, this is actually our fault, we still haven't put an ESLint file, but could you make sure that the spacing is right everywhere in the PR (i.e. `{ message`)
thanks!

File: src/index.js [hunk 3]
```diff
@@ -218,6 +226,24 @@ module.exports = {
...
+      })
+
+      router.post("/remove_payment_tester", (req, res) => {
+        const prmse = messenger.deletePaymentTester(req.body.payment_tester)
```
Comment (slvnperron) [inline src/index.js]: looks like the promise isn't used?

**Comment by Jalepeno112** [comment]
For all of these issues you've pointed out, I made a really dumb `rebase` yesterday and a lot of things I thought I deleted came back.  I'll clean this up and get back to you!

**Comment by Jalepeno112** [comment]
@slvnperron I made the changes and rebased yesterday!  I'm in no rush, but wanted to make sure you saw it.  Thanks!

**Comment by slvnperron** [comment]
@Jalepeno112 awesome!

**Comment by slvnperron** [comment]
Just pushed to `botpress-messenger@2.0.14`. Thanks again for the PR!

**Pull request merged**